purchase_work_acceptance_late_fine/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    wa_fine_rate = fields.Monetary(string="Rate Fine")
+    wa_late_fine_account_id = fields.Many2one(
+        comodel_name="account.account", string="Late Fine Revenue Account",
```

Comment:
```suggestion
        comodel_name="account.account", string="Late Delivery Fines Revenue Account",
```

Reply:
And all others...

Reply:
Currently, system will use "Accept Date" - "Due Date" to calculate late days
![image](https://user-images.githubusercontent.com/1973598/99058976-7e75b280-25d0-11eb-86aa-aba9dec3acc5.png)

Accept date = date when click the button. I think this is very strict. Also once accepted, user can't change the late days or rated / day.

I think we should make late days and rate / day as editable even Accepted. But make sure all changes are logged in Chatter as Track=True.

Reply:
![image](https://user-images.githubusercontent.com/1973598/99059551-55a1ed00-25d1-11eb-9d2a-0d9b3b035cf0.png)

1. Create and Edit should be disable here.
2. Select WA, should auto fill Partner. (if user select partner first, also filer only WA of partner).